architektura_platformy.rst

```diff
@@ -0,0 +1,60 @@
+﻿Architektura platformy aplikacje.gov.pl - szkic
+===============================================
+
+Cel
+---
+
+Odpowiedź na pytanie „W jakim języku programowania pisany jest system, jakiej bazy danych używa i czy dostępne są materiały opisujące architekturę rozwiązania.”
+
+Opisanie szkicu architektury platformy aplikacje.gov.pl
+
+Krótka odpowiedź
+----------------
+
+* architektura: mikroserwisy
+* język programowania: dowolny, uruchamialny w środowisku linuksowym
+* baza danych: dowolna, uruchamialna w środowisku linuksowym, najlepiej w architekturze serwer-klient
+
+Mikroserwisy
+------------
+
+Moduły (serwisy) działające w obrębie platformy będą oddzielnymi programami. Między sobą będą komunikować się przez sieć, np przez HTTP (inne protokoły też są dopuszczone). Dla przykładu relacyjna baza danych też będzie serwisem, do którego inne serwisy będą mogły się połączyć specyficznym dla niej protokołem.
+
+Serwisy będą uruchamiane w środowisku linuksowym.
+
+Pisanie kodu nowych serwisów to pisanie programów. Język jest dowolny, o ile można go uruchomić na platformie linuksowej. Kompilacja może się odbywać na platformie nie-linuksowej. Serwis może korzystać z innych serwisów - ich adresy powinny być konfigurowalne (np. w pliku konfiguracyjnym lub w wywołaniu linii poleceń). Program powinien mieć możliwość przyjmowania tych adresów jako nazw hostów i dekodowania ich zapytaniami DNS, ponieważ adresacja w sieci może być dynamiczna.
```

Comment:
„nielinuksowej”, chyba że traktujemy przymiotnik jako dzierżawczy, to wtedy „Linuksowej” i „nie-Linuksowej”.